api/utils.js

```diff
@@ -0,0 +1,8 @@
+export const getSkillWalletNonce = async () => {
+    const response = await fetch('https://api.skillwallet.id/api/skillwallet/-1/nonces', {
```

Comment:
this request returns 400 and the encoded string is "skillWalletId is required." 
Please add query param action=1
https://api.skillwallet.id/api/skillwallet/-1/nonces?action=1